### ISSUE TITLE
Investigate use of H3's in terms of accessibility in OJ's

### ISSUE THREAD

**Issue by mykeels** [bbc/simorgh#5793]
**Describe the bug**
CPS Related Content and Features & Analysis currently use H3 tags even though there is no relevant text content which could be a potential accessibility issue. See [Conversation](https://github.com/bbc/simorgh/pull/5751#pullrequestreview-368125420). Discuss with @greenc05 for clarification on what needs to be done. We may need to modify the psammead component then renders these components. This may also have an impact on the use of the headlines on other page types like front pages, speak to the ws-home team as this will affect them.

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to https://www.test.bbc.com/pidgin/23259748
2. Scroll down to '....'
3. Right-Click on any link title in the grid below, and Inspect Element.
5. Note that it's an H3

**Expected behaviour**
- TBC

**Screenshots**

![image](https://user-images.githubusercontent.com/11996508/76084826-171f5180-5fb1-11ea-93a4-b78ebf9465f6.png)

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

**Comment by greenc05** [comment]
Ok, so apologies if my use of the words 'related content' confused things here. If there is no content below the promo title, then the title should not be a heading, as headings should have related content below them. So in the case of F&A, each li would not have any content below the 'title'. However in the 'Related content' component we have the time stamp below the promo title/heading so technically this is fine as a heading https://github.com/bbc/bbc-a11y/blob/master/lib/standards/tests/contentMustFollowHeadings.js

The only other exception that I'm aware of is the onwards journey component in the middle of the article/story body text, this component cannot have any headings at all due to being in the article/story body and would head up all the article/story content below it, which would not be accurate

All other promos should stay as they are please.

**Comment by greenc05** [comment]
This can be closed, there was a mix up with requirements, each F&A (and related content) promo will always have content below the h3, a timestamp on mobile, and on other breakpoints there will be a summary and timestamp
